packages/rancher-monitoring/rancher-monitoring/package.yaml

```diff
@@ -1,7 +1,7 @@
 url: https://github.com/prometheus-community/helm-charts.git
 subdirectory: charts/kube-prometheus-stack
 commit: 7c6906ca223344c06952007fda670c6c81e6d1da
-version: 104.1.1
+version: 104.1.2-rc.1+up57.0.3
```

Comment:
The upstream version is added by `chart-build-scripts` change to `104.1.2-rc.1` thanks!